hw/ip/otbn/dv/uvm/gen-binaries.py

```diff
@@ -61,60 +61,29 @@ def run(self, cmd: List[str]) -> None:
         subprocess.run(cmd, env=env)
 
 
-def take_env_line(dst: Dict[str, str], path: str, line_number: int,
-                  line: str) -> None:
-    '''Read one line from a .env file, updating dst.'''
-    line = line.split('#', 1)[0].strip()
-    if not line:
-        return
+def read_toolchain(otbn_dir: str) -> Toolchain:
+    '''Reads environment variables to get toolchain info.'''
```

Comment:
Maybe this should be something like `get_toolchain` now? Since it's no longer reading a file.

Reply:
Fixed.